README.md

```diff
@@ -106,7 +106,7 @@ $args = array(
       'draft',  // - a post in draft status.
       'auto-draft', // - a newly created post, with no content.
       'future', // - a post to publish in the future.
-      'private', // - not visible to users who are not logged in.
+      'private', // - not visible to any users.
```

Comment:
Whoops - I wasn't clear. Please remove this one too.

Reply:
Logged in user also can not see this post. That's why I changed it a bit.

Reply:
Only the author of the post can see this post ☺

Reply:
Private content is visible to `those with authorization permission levels to see private content`. By default this includes Editors and Administrators. https://wordpress.org/support/article/content-visibility/#private-content

How about changing line 109 to:
`'private', // - visible only to the author, administrators, and editors.`

Reply:
Yes, It could be better :slightly_smiling_face: